### DISCUSSION TITLE
Moving repositories to another organisation

### DISCUSSION THREAD

**Discussion by pgundlach** [speedata/boxesandglue#8] [Announcements]
I will move the main repositories to the new organisation (https://github.com/boxesandglue) in a few days. I hope that this will not break anything...

I someone has experience with such a location change, please comment what I should take into account.

**Comment by alerque** [comment]
I've done this lots. It's surprisingly easy, just do the transfer and GH will take care of forwarding almost everything to the new locations.

The big gotcha is that you should never setup anything at the old names or *fork* them back from the new canonical location to the old one. If you do that then you loose all the nice automatic URL rewriting and forwarding going on and suddenly people looking for the old resources will end up on the wrong fork or even entirely the wrong resource. And you don't get to set them up a second time, it's one shot when you first transfer the repo(s).

**Reply by pgundlach** [reply]
Thank you for the reply, very much appreciated! I should not be too much afraid of this step.
